stories/module-analytics-settings.stories.js

```diff
@@ -425,146 +410,4 @@ storiesOf( 'Analytics Module/Settings', module )
 		{
 			decorators: [ WithRegistry ],
 		}
-	)
-	.add(
-		'Edit, with existing tag w/ access',
-		( args, { registry } ) => {
-			const { dispatch } = registry;
-			const {
-				accounts,
-				properties,
-				profiles,
-				matchedProperty,
-			} = accountsPropertiesProfiles;
-			const existingTag = {
-				// eslint-disable-next-line sitekit/acronym-case
-				accountID: matchedProperty.accountId,
-				propertyID: matchedProperty.id,
-			};
-
-			dispatch( MODULES_ANALYTICS ).receiveGetAccounts( accounts );
-			dispatch( MODULES_ANALYTICS ).receiveGetProperties( properties, {
-				accountID: properties[ 0 ].accountId,
-			} ); // eslint-disable-line sitekit/acronym-case
-			dispatch( MODULES_ANALYTICS ).receiveGetProfiles( profiles, {
-				accountID: properties[ 0 ].accountId, // eslint-disable-line sitekit/acronym-case
-				propertyID: profiles[ 0 ].webPropertyId, // eslint-disable-line sitekit/acronym-case
-			} );
-			dispatch( MODULES_ANALYTICS ).receiveGetSettings( defaultSettings );
-			dispatch( MODULES_ANALYTICS ).receiveGetExistingTag(
-				existingTag.propertyID
-			);
-
-			dispatch( MODULES_ANALYTICS_4 ).receiveGetProperties(
-				[
-					{
-						_id: '1001',
-						displayName: 'GA4 Property',
-					},
-				],
-				{ accountID: existingTag.accountID }
-			);
-
-			return (
-				<Settings
-					registry={ registry }
-					route="/connected-services/analytics/edit"
-				/>
-			);
-		},
-		{
-			decorators: [ WithRegistry ],
-		}
-	)
-	.add(
-		'Edit, with existing tag w/o access',
```

Comment:
As far as I could tell, this was the equivalent of the **Existing Tag w/o access story** listed in the AC so I deleted it.